.travis.yml

```diff
@@ -1,2 +1,23 @@
 language: d
+dist: trusty
 sudo: false
+
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
```

Comment:
Heh, I rediscovered why the long form was used here: & and [ are special characters, at least when they appear at the end of lines.
If I escape that stuff, the YAML is actually harder to read, as the formatting of every line in the listing has to be changed (or everything needs to be set in quotes alternatively).